src/hotspot/share/oops/methodData.hpp

```diff
@@ -1088,21 +1088,8 @@ class ReceiverTypeData : public CounterData {
   friend class JVMCIVMStructs;
 protected:
   enum {
-#if INCLUDE_JVMCI
```

Comment:
I think you should add a comment to ReceiverTypeData explaining when count is incremented